### ISSUE TITLE
login.py fails

### ISSUE THREAD

**Issue by michardy** [BaySchoolCS2/ProjectRepo#3]
``` python
michael@michael-HP-EliteBook-8470p ~/Code/CS2/ProjectRepo $ ./run.py 
Traceback (most recent call last):
  File "./run.py", line 3, in <module>
    from application import app
  File "/home/michael/Code/CS2/ProjectRepo/application/__init__.py", line 8, in <module>
    from application import login, home, signup
  File "/home/michael/Code/CS2/ProjectRepo/application/login.py", line 11
    user = User(email=)
                      ^
SyntaxError: invalid syntax
```

**Comment by fhebert-perkins** [comment]
I'M NOT WRITING THE WHOLE BLOODY THING, all i said was that i had gotten it to a point where other people could finish it... There is plenty of example code out there... this is just so that we all dont go insane later when we try to change things... building conventions you know

**Comment by fhebert-perkins** [comment]
Fixed as of #4. I just went ahead and wrote the "WHOLE BLOODY THING". Turns out i started it completely wrong :+1: 
That should serve as a working example of how to interface with the database. Closing this one out.
